test/integration/integration-test.js

```diff
@@ -147,10 +175,145 @@ describe.skip('integration tests', function() {
     ], () => done());
   });
 
+
+  it('isconnected', function(done) {
```

Comment:
No. orderSpecification has field immediateOrCancel, and createdOrder.specification - doesn't.